src/modules/fancyzones/lib/WindowMoveHandler.cpp

```diff
@@ -56,11 +57,13 @@ namespace WindowMoveHandlerUtils
 class WindowMoveHandlerPrivate
 {
 public:
-    WindowMoveHandlerPrivate(const winrt::com_ptr<IFancyZonesSettings>& settings, SecondaryMouseButtonsHook* mouseHook, ShiftKeyHook* shiftHook, CtrlKeyHook* ctrlHook) :
+    WindowMoveHandlerPrivate(const winrt::com_ptr<IFancyZonesSettings>& settings, const std::function<void()>& keyUpdateCallback) :

```

Comment:
callback could even be defined inside the constructor.. not necessary to be passed as arg

Reply:
You mean to call directly `WindowMoveHandler::MoveSizeUpdate`?